doc/topics/releases/2015.8.10.rst

```diff
@@ -5,6 +5,12 @@ Salt 2015.8.10 Release Notes
 Version 2015.8.10 is a bugfix release for :doc:`2015.8.0
 </topics/releases/2015.8.0>`.
 
+**Final Release of Debian 7 Packages**
```

Comment:
@jacobhammons Can you add this notice to the 2016.3.1 release notes in the 2016.3 branch as well?